assets/create.sh

```diff
@@ -2,6 +2,43 @@
 set -euo pipefail
 
 ASSET_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+REPO_DIR="$ASSET_DIR/.."
+
+# Ensure submodules are initialized.
+# This might look like black magic, but it works.
```

Comment:
I think we can remove this line.

Reply:
Oops. I forgot to remove that before committing. Yeah, I'll fix that.